src/widgets/input/MultiInputWidget.js

```diff
@@ -0,0 +1,122 @@
+// @flow
+import React, { useState } from 'react';
+
+import styled from 'styled-components';
+import { Input } from 'lattice-ui-kit';
+
+import type { WidgetProps } from '../../../types';
+
+const InputWrapper = styled.div`
+  display: grid;
+  grid-template-columns:${(props) => `${props.flexOptions[0]}fr ${props.flexOptions[1]}fr`};
+  grid-gap: 20px;
+  margin-bottom: 15px;
+  align-items: center;
+
+  >span {
+    font-size: 14px;
+  }
+`;
+
+const MultiInputWidget = (props :WidgetProps) => {
+
+  const {
+    autofocus,
+    disabled,
+    formContext,
+    id,
+    onBlur,
+    onFocus,
+    options,
+    rawErrors,
+    readonly,
+    registry,
+    schema,
+    value,
+    onChange,
+    ...inputProps
+  } = props;
+
+  inputProps.type = 'text';
+
+  if (options.inputType) {
+    inputProps.type = options.inputType;
+  }
```

Comment:
I get the sense that you're doing this strange override to get around some errors that the form might be throwing. Why not just let the inputProps.type just be what is set in the schema?